stage_descriptions/sorted-sets-03-lg6.md

```diff
@@ -0,0 +1,77 @@
+In this stage, you'll add support for retrieving the rank of a sorted set member.
+
+### The `ZRANK` Command
+
+The `ZRANK` command is used to query the rank of a member in a sorted set. It returns an integer, which is 0-based index of the member when the sorted set is ordered by increasing score.
+If two members have same score, the members are ordered lexicographically.
+
+Example usage:
+```bash
+> ZADD zset_key 1.0 member_with_score_1
+(integer) 1
+> ZADD zset_key 2.0 member_with_score_2
+(integer) 1
+> ZADD zset_key 2.0 another_member_with_score_2
+(integer) 1
+
+
+> ZRANK zset_key member_with_score_1
+(integer) 0
+> ZRANK zset_key member_with_score_2
+(integer) 2
+> ZRANK zset_key another_member_with_score_2
+(integer) 1
+```
+
+The rank of `another_member_with_score_2` is 1, and `member_with_score_2` is 2. It is because though the both members have same scores, `another_member_with_score_2` preceeds `member_with_score_2` lexicographically.
+
+
+If the member, or the sorted set does not exist, the command returns null bulk string.
```

Comment:
Users might not recall what "null bulk string" is - look at how we mention this in other stages, stay consistent